src/Modules/Meetings/Application/MeetingComments/GetMeetingCommentLikers/GetMeetingCommentLikersQueryHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using CompanyName.MyMeetings.BuildingBlocks.Application.Data;
+using CompanyName.MyMeetings.Modules.Meetings.Application.Configuration.Queries;
+using Dapper;
+
+namespace CompanyName.MyMeetings.Modules.Meetings.Application.MeetingComments.GetMeetingCommentLikes
+{
+    internal class GetMeetingCommentLikersQueryHandler : IQueryHandler<GetMeetingCommentLikersQuery, List<MeetingCommentLikerDto>>
+    {
+        private readonly ISqlConnectionFactory _sqlConnectionFactory;
+
+        public GetMeetingCommentLikersQueryHandler(ISqlConnectionFactory sqlConnectionFactory)
+        {
+            _sqlConnectionFactory = sqlConnectionFactory;
+        }
+
+        public async Task<List<MeetingCommentLikerDto>> Handle(GetMeetingCommentLikersQuery query, CancellationToken cancellationToken)
+        {
+            var connection = _sqlConnectionFactory.GetOpenConnection();
+
+            var sql = "SELECT " +
+                      "[Liker].[Id]," +
+                      "[Liker].[Name] " +
```

Comment:
Please use `nameof(MeetingCommentLikerDto.PropertyName)` for each returned property

Reply:
fixed